macros/src/parser/state_machine.rs

```diff
@@ -13,6 +15,8 @@ impl StateMachine {
         StateMachine {
             temporary_context_type: None,
             custom_guard_error: false,
+            impl_display_states: false,
+            impl_display_events: false,
```

Comment:
I'd make these default-enabled. Unused code will just get stripped by the compiler.

Reply:
This would break projects where authors manually implemented these thought.
Maybe we should enable those by default on a v1.0.